src/components/multi-action-button/multi-action-button.js

```diff
@@ -3,10 +3,10 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Icon from '../icon';
 import Button from '../button';
-import SplitButton from '../split-button';
+// import SplitButton from '../split-button';
 import './multi-action-button.scss';
 
-class MultiActionButton extends SplitButton {
+class MultiActionButton extends React.Component {
   static propTypes = {
```

Comment:
Maybe it will be good to move propTypes to bottom section of the file? I mean

```js 
MultiActionButton.propTypes {}
MultiActionButton.defaultProps{}
```

Reply:
Yeah now we have an agreed way I'll change them